cryptography/src/bls12381/scheme.rs

```diff
@@ -359,6 +425,39 @@ impl Display for Signature {
 mod tests {
     use super::*;
 
+    #[test]
```

Comment:
If we agree to "feature freeze" the old code, I wonder if it makes sense to just update the existing tests with the new logic?

Unless you were following a particular approach here?

Reply:
I'm afraid to cut a release that bypasses all those tests (if we migrate crates to run on these).

Dumb idea would just be to clone all the tests but rename them for this new stuff (then drop the old ones once we remove).